apps/extension/src/entrypoints/host.content/style.css

```diff
@@ -68,3 +68,12 @@
     -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Yu Gothic', 'YuGothic', 'Hiragino Kaku Gothic ProN', 'Meiryo',
     'MS PGothic', 'MS Gothic', 'Noto Sans JP', 'Source Han Sans JP', 'IPAexGothic', Arial, Helvetica, sans-serif !important;
 }
+
+.read-frog-translated-content-wrapper[dir='rtl'] {
```

Comment:
fonts from `lang` and `dir` have the conflicts, then what will happend?

Reply:
Among the languages currently supported by `franc-min`, all RTL scripts use the Arabic alphabet, so they can share the same font set without conflict. Only if we add support for Hebrew or even traditional Mongolian in the future will potential conflicts between two font families need to be considered.

At that point, the font-family currently defined under rtl can be removed and replaced with a dedicated font-family setting for Arabic, using the same approach as the current Chinese and Japanese font configuration.